tensorflow_addons/image/tests/filters_test.py

```diff
@@ -18,6 +18,9 @@
 import tensorflow as tf
 from tensorflow_addons.image import mean_filter2d
 from tensorflow_addons.image import median_filter2d
+from tensorflow_addons.image import gaussian_filter2d
+from scipy.ndimage import gaussian_filter
+from skimage.filters import gaussian
```

Comment:
Though it does not matter, I wonder why we use two different implementations to test ours. Can we just use one of them?